app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If your teacher training application is unsuccessful
+description: |-
+  Read about what to do if your teacher training application is unsuccessful. Find advice on why an application can be unsuccessful and how to get support.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
+    7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
+    11 top tips for interview success : "/blog/11-top-tips-for-interview-success"
+date: "2023-03-30"
+keywords:
+  - adviser
+  - advisor
+  - ITT
+  - Initial Teacher Training
+  - QTS
+  - Qualified Teacher Status
+  - Teacher Training Adviser
+  - application
+  - apply
+  - how to apply
+  - applying for teacher training
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+---
+
+Try not to worry - you can reapply for up to 4 courses at a time, as many times as you want. Many people are successful the second or third time around.
+
+It’s worth looking at the feedback the course provider has given to understand why your application was not accepted. This will help you to decide on your next steps.
+
+## Why was my application unsuccessful?
+
+There could be a number of reasons your application was unsuccessful:
+
+### The course is full
+
+Unfortunately, popular courses do fill up quickly. You can [search for other courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) with a different provider if the one you applied for is now full.
+
+### You do not have the GCSEs at minimum grade 4 (C), or equivalent
+
+To train to teach in primary and secondary schools in England, you’ll need:
+
+* GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)
+
+You could [find another course](https://www.gov.uk/find-postgraduate-teacher-training-courses) with a training provider that will accept an equivalency test. You can also take your GCSE exams at any age if you do not have them or you can retake them to improve your grades.
```

Comment:
Should we signpost people to how to take/retake GCSEs here?

On the adviser funnel dead end page about GCSEs, we say the following:

`If you do not have the right GCSEs, you can get them through a [GCSE training provider](https://nationalcareers.service.gov.uk/find-a-course/search). Make sure when you visit the website, you search for GCSE courses.`

Appreciate this adds more links so happy for you to leave this out if needed